tests/unit/helpers/date-format-test.js

```diff
@@ -0,0 +1,10 @@
+import { momentFormat } from '../../../helpers/moment-format';
+import { module, test } from 'qunit';
+
+module('Unit | Helper | moment format');
+
+// Replace this with your real tests.
+test('it works', function(assert) {
+  var result = momentFormat(42);
```

Comment:
and one last `dateFormat([new Date], {format:'M/D/YYYY'})` or something like that i think